### PR TITLE
Add feature flag for fetching master branch runs only

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Adds a field to the query builder, and uses the `label=master` in the default query.

## Review Information
- Head to /flags, enable the feature
- Open the query builder, tick the box, reload, see `master` label is used

### CODE REVIEW

File: webapp/components/test-runs-query.html [hunk 1]
```diff
@@ -72,11 +73,15 @@
...
 
-        computeTestRunQueryParams(sha, aligned, labels, productSpecs, maxCount, from, to) {
+        computeTestRunQueryParams(sha, aligned, master, labels, productSpecs, maxCount, from, to) {
           const params = {};
           if (!this.computeIsLatest(sha)) {
             params.sha = sha;
           }
+          if (this.masterRunsOnly && master
+              && (!labels || !labels.includes('master'))) {
+            labels = (labels || []).concat('master');
```
Comment (mdittmer) [inline webapp/components/test-runs-query.html]: This should be `labels = (labels || []).concat(['master']);` or `labels = labels || []; labels.push('master');`

File: webapp/components/test-runs-query.html [hunk 2]
```diff
@@ -182,6 +187,13 @@
           if ('aligned' in params) {
             batchUpdate.aligned = params.aligned;
           }
+          if (this.masterRunsOnly) {
```
Comment (mdittmer) [inline webapp/components/test-runs-query.html]: I'm confused. Cica line 81 we model "master only" as a label, `master`. Here, we're modeling it as a property, `master:true`, and removing its corresponding label. Why don't we model it consistently one way or the other?

File: webapp/components/test-runs-query.html (hunk 1 above)
Comment (lukebjerring) [reply webapp/components/test-runs-query.html]: We don't want to mutate the UI property (potentially there was a bug before this PR). Your code is more readable though; changed.

File: webapp/components/test-runs-query.html (hunk 2 above)
Comment (lukebjerring) [reply webapp/components/test-runs-query.html]: Server-side URI params vs "less confusing UI" properties.

**Comment by lukebjerring** [comment]
No, the changes were the same treatment of labels as a copy of the array (and always existing now).

**Pull request merged**